CMakeLists.txt

```diff
@@ -821,3 +826,32 @@ IF(CPUINFO_SUPPORTED_PLATFORM AND CPUINFO_BUILD_TOOLS)
     INSTALL(TARGETS cpuid-dump RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
   ENDIF()
 ENDIF()
+
+# ---[ pkg-config manifest. This is mostly from JsonCpp...
```

Comment:
Make sure the code style is consistent with the rest of cpuinfo's CMakeLists

Reply:
Done. Have a look.

Reply:
Please capitalize the CMake command names and use the same nested indentation as the rest of the file.

Reply:
Fixed the case. Indentation looks the same as elsewhere, unless I missed something. Feel free to edit.